core/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -7,6 +7,20 @@
 
   <include layout="@layout/layout_standard_app_bar" />
 
+  <TextView
+    android:id="@+id/no_bookmarks"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/no_bookmarks"
+    android:textSize="15sp"
```

Comment:
we have a typography system, see `themes.xml`
```
 <!--Material type attributes-->
    <item name="textAppearanceHeadline1">@style/TextAppearance.KiwixTheme.Headline1</item>
    <item name="textAppearanceHeadline2">@style/TextAppearance.KiwixTheme.Headline2</item>
    <item name="textAppearanceHeadline3">@style/TextAppearance.KiwixTheme.Headline3</item>
    <item name="textAppearanceHeadline4">@style/TextAppearance.KiwixTheme.Headline4</item>
    <item name="textAppearanceHeadline5">@style/TextAppearance.KiwixTheme.Headline5</item>
    <item name="textAppearanceHeadline6">@style/TextAppearance.KiwixTheme.Headline6</item>
    <item name="textAppearanceSubtitle1">@style/TextAppearance.KiwixTheme.Subtitle1</item>
    <item name="textAppearanceSubtitle2">@style/TextAppearance.KiwixTheme.Subtitle2</item>
    <item name="textAppearanceBody1">@style/TextAppearance.KiwixTheme.Body1</item>
    <item name="textAppearanceBody2">@style/TextAppearance.KiwixTheme.Body2</item>
    <item name="textAppearanceButton">@style/TextAppearance.KiwixTheme.Button</item>
    <item name="textAppearanceCaption">@style/TextAppearance.KiwixTheme.Caption</item>
    <item name="textAppearanceOverline">@style/TextAppearance.KiwixTheme.Overline</item>
```
choose the one most appropriate for this textview and then there is no need to specify the text size in the layout

Reply:
it can be applied like so `android:textAppearance="?textAppearanceHeadline6"`